packages/notifi-react-card/lib/hooks/useIntercomChat.ts

```diff
@@ -52,6 +55,35 @@ export const useIntercomChat = ({
   const [isScrolling, setIsScrolling] = useState<boolean | null>();
   const { client } = useNotifiClientContext();
 
+  useEffect(() => {
+    const intervalId = setInterval(function () {
+      client
+        .getConversationMessages({
+          first: 50,
+          getConversationMessagesInput: { conversationId },
+        })
+        .then((response) => {
+          if (Array.isArray(response.nodes)) {
+            const nodes = response.nodes;
+            if (nodes[0].id === chatMessages[0].id) {
+              return;
+            } else {
+              const chatMessageIds = chatMessages.map((message) => message.id);
+              const dedupeNewMessages = nodes.filter(
+                (node) => !chatMessageIds.includes(node.id),
+              );
```

Comment:
it would be nice if chatMessageIds were a Set instead (using `reduce` instead of `map`) to improve runtime of `includes` check